admin/src/scenes/dashboard-head-center/center/status.js

```diff
@@ -13,10 +13,21 @@ export default function Status() {
 
   useEffect(() => {
     (async () => {
-      const { data, ok, code } = await api.get(`/cohesion-center/${user.cohesionCenterId}`);
-      if (!ok) return console.log(code);
-      setPlacesTotal(data.placesTotal);
-      setPlacesLeft(data.placesLeft);
+      const bodySession = {
+        query: { bool: { must: { match_all: {} }, filter: [{ terms: { cohesionCenterId: [user.cohesionCenterId] } }] } },
```

Comment:
un chef de centre peut etre lié a plusieurs sessions, il faudrait pas prendre le truc dans l'autre sens ducoup ? en fait on aurait pas du foutre cette double liaison user <-> session
```suggestion
        query: { bool: { must: { match_all: {} }, filter: [{ term: { headCenterId: user._id } }] } },
```